acceptance: TestDockerCLI failed

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1785662&tab=buildLog) on [master@1e97ffbd854b2a7fdb7e3a9376b627fa0c28c216](https://github.com/cockroachdb/cockroach/commits/1e97ffbd854b2a7fdb7e3a9376b627fa0c28c216):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (350.45s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI074909913
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (62.44s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1797727&tab=buildLog) on [master@a99194d5933b900b7e84e19d37996ddda55874cb](https://github.com/cockroachdb/cockroach/commits/a99194d5933b900b7e84e19d37996ddda55874cb):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (365.45s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI332726236
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (62.50s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1855392&tab=buildLog) on [master@bf8e3784fa4ea561b1714b8d34840750ec80d49d](https://github.com/cockroachdb/cockroach/commits/bf8e3784fa4ea561b1714b8d34840750ec80d49d):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (318.73s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI187455077
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (9.46s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cli/interactive_tests/test_demo_partitioning.tcl
+++ b/pkg/cli/interactive_tests/test_demo_partitioning.tcl
@@ -15,7 +15,7 @@ expect {
   # can fail in various ways (i.e. not just a timeout).
   # So instead we use the specific error message shouted by `cockroach demo` when license
   # acquisition fails.
-  "license acquisition was unsuccessful" {
+  "unable to acquire a license for this demo" {
       # The license server is unreachable. There's not much we can test here.
       # Simply ignore the test.
       report "License server could not be reached - skipping with no error"

